src/workloads/contrib/qe_test_gen/patchConfig.yml

```diff
@@ -40,39 +40,6 @@ patches:
       - medical_workload_guid_50_50_unencrypted
       - medical_workload_guid_50_50_unencrypted
       - medical_workload_guid_50_50_unencrypted
-    linux-shard-lite-fle.2022-11:
```

Comment:
It might be better to replace this variant with it's replacement [perf-3-shard-fle.arm.aws.2023-11](https://github.com/10gen/dsi/blob/master/evergreen/system_perf/master/variants.yml#L335C11-L335C43).